spec/forms/two_factor_setup_form_spec.rb

```diff
@@ -56,26 +57,30 @@
       it 'is valid' do
         user = build_stubbed(:user, phone: valid_phone)
         form = TwoFactorSetupForm.new(user)
-
-        result = {
-          success: true,
-          error: nil,
+        extra = {
           otp_method: 'sms',
         }
+        result = instance_double(FormResponse)
 
+        expect(FormResponse).to receive(:new).
```

Comment:
I didn't think it was necessary here. Here, we're mainly testing the validation logic. The fact that the return value is a FormResponse is tested in the expectations. I'm not sure we need to include that in the description as well.